ui/src/shared/components/ConfirmButtons.js

```diff
@@ -1,11 +1,22 @@
 import React, {PropTypes} from 'react'
+import classnames from 'classnames'
 
-const ConfirmButtons = ({onConfirm, item, onCancel}) => (
+const ConfirmButtons = ({onConfirm, item, onCancel, buttonSize}) => (
   <div className="confirm-buttons">
-    <button className="btn btn-xs btn-info" onClick={() => onCancel(item)}>
+    <button
+      className={classnames('btn btn-info btn-square', {
+        [buttonSize]: buttonSize,
+      })}
```

Comment:
I had a feeling there was a cleaner way to do this. My intent is to add the value of `buttonSize` only if it was passed in as a prop. That being said it has a default value so it would never be null. Might make more sense to do a template string as the first argument of `classnames`